static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -1,66 +1,47 @@
-/* jscs:disable disallowDanglingUnderscores */
 define([
-    'common/utils/$',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'Promise',
     'raven',
     'common/modules/article/spacefinder'
 ], function (
-    $,
-    idleFastdom,
+    fastdom,
```

Comment:
The question is, when is it no longer appropriate to use the name fastdom? The proxy we're looking at here is fastdom wrapped in an idle callback wrapped in a promise interface. Should the name reflect all those details?

I wonder if we should just say, 'this is the Guardian proxy for writing to the DOM, it'll do things as fast as possible but at the cost of synchronicity, and that's our interface'. I tend to think that dropping the name 'fastdom' will make developers feel more free to make the proxy thicker - they are less bound to strictly follow the behaviour of fastdom, so they can add more optimizations and filters. This may or may not be a good thing, and may not be necessary yet in any case.

The more I think of it, the less I like the idea of a big, thick proxy. So perhaps keeping the word 'fastdom' in our name is a subtle way to keep the wrapper thin, at least for the moment.